.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
One note -- it's often not good practice to commit files specific to your IDE / configuring your IDE (I appreciate VSCode may be the gold standard in the KOL scripting community though, so feel free to ignore this thought).

Reply:
On the one hand, valid. On the other, there's vscode settings in the current version of the repo (and in garbo, which is where I stole the tasks from), and if scotch and bean think they should be there, I am not gonna argue.

Reply:
totally fair -- and your repo your rules obviously! -- was just speaking as a Sublime user who would remove all this stuff as my first commit (honestly scotch and bean should consider taking it out of their repo as well; if it is required for the project to function it should be IDE agnostic!)